src/components/KonnectorSuccess.jsx

```diff
@@ -94,7 +92,7 @@ KonnectorSuccess.apps = {
       return res
     },
     // eslint-disable-next-line react/display-name
-    successLink: (props, context, i) => {
+    successLink: (props, i) => {
```

Comment:
We should also update our override method

Reply:
We have to be careful here here and check that the BankLink and DriveLink still work.

Reply:
I dropped the commit and add comments to prevent that it used for customisation instead, see f69b084f